runtime/queries/tact/highlights.scm

```diff
@@ -0,0 +1,210 @@
+; NOTE: Order of highlight queries matters, as Tree-sitter uses the first it finds.
+; NOTE: Therefore, narrow highlight queries should be placed before broad captures.
+; ---------------------------------------------------------------------------------
+
+; attribute
+; ---------
+
+[
+  "@name"
+  "@interface"
+] @attribute
+
+; comment
+; -------
+
+(comment) @comment
+
+; function.builtin
+; ----------------
+
+((identifier) @function.builtin
+  (#match? @function.builtin
+     "^(send|sender|require|now|myBalance|myAddress|newAddress|contractAddress|contractAddressExt|emit|cell|ton|dump|beginString|beginComment|beginTailString|beginStringFromBuilder|beginCell|emptyCell|randomInt|random|checkSignature|checkDataSignature|sha256|min|max|abs|pow|throw|nativeThrowWhen|nativeThrowUnless|getConfigParam|nativeRandomize|nativeRandomizeLt|nativePrepareRandom|nativeRandom|nativeRandomInterval|nativeReserve)$")
+  (#is-not? local))
+
+; function.method
+; ---------------
+
+(method_call_expression
+  name: (identifier) @function.method)
+
+; function
+; --------
+
+(func_identifier) @function
+
+(native_function
+  name: (identifier) @function)
+
+(static_function
+  name: (identifier) @function)
+
+(static_call_expression
+  name: (identifier) @function)
+
+(init_function
+  "init" @function.method)
+
+(receive_function
+  "receive" @function.method)
+
+(bounced_function
+  "bounced" @function.method)
+
+(external_function
+  "external" @function.method)
+
+(function
+  name: (identifier) @function.method)
+
+; keyword
+; -------
+
+[
+  "get" "mutates" "extends" "virtual" "override" "inline" "abstract"
+  "contract" "trait" "struct" "message" "with"
+  "const" "let" "fun" "native"
+  "primitive" "import"
+  "if" "else" "while" "repeat" "do" "until"
+  "return" "initOf"
+  ; "public" ; -- not used, but declared in grammar.ohm
+  ; "extend" ; -- not used, but declared in grammar.ohm
+] @keyword
+
+; number
+; ------
+
+(integer) @number
```

Comment:
Helix uses a custom set of captures, see https://docs.helix-editor.com/master/themes.html#syntax-highlighting

So this should be `@constant.numeric.integer`, `@property` should be `@variable.other.member`, etc..